wcfsetup/setup/db/install.sql

```diff
@@ -1432,6 +1432,19 @@ CREATE TABLE wcf1_trophy(
 	KEY(categoryID)
 );
 
+DROP TABLE IF EXISTS wcf1_unfurl_url;
+CREATE TABLE wcf1_unfurl_url (
+	urlID INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	url TEXT NOT NULL,
+	urlHash VARCHAR(40) NOT NULL UNIQUE KEY,
```

Comment:
Make this an explicit `UNIQUE KEY (urlHash)` at the end of the table definition.